projectceladon/vendor-intel-utils
# Remove BT HAL bsp_diff patches

Remove BT HAL bsp_diff patches and merge to opensource repo.

Tests done:
1. Flash binary
2. BT on/off success
3. #>adb shell cat proc/asound/cards
4. btaudiosource listed
5. Connect BT headset
6. Verify sco handle sent from bt-hal

Tracked-On: OAM-126126
Signed-off-by: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
